docs/view/css.md

```diff
@@ -53,3 +53,31 @@ Media breakpoints instruct the browser to apply different CSS depending on the v
 | `screen-md-down` | Smartphones and Tablets | `(max-width: 1024px)` |
 | `screen-md-up` | Tablets (landscape) and desktop PC | `(min-width: 769px)` |
 | `screen-lg` | Desktop PC | `(min-width: 1025px)` |
+
+## Asset Preloading
+
+WoltLab Suite’s SCSS compiler supports adding [preloading](https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content) metadata to the CSS.
+To communicate the preloading intent to the compiler the `--woltlab-suite-preload` CSS variable is set to the result of the `preload()` function:
+
+```scss
+body {
```

Comment:
I would use a different selector (`.fooBar`) to emphasize that this is an example and no existing code.